test/parser-test.cpp

```diff
@@ -813,7 +815,7 @@ TEST(serializationTests, serializationTests2) {
   char * buffer = parser.serialize(&size);
 
   ABPFilterParser parser2;
-  parser2.deserialize(buffer);
+  CHECK(parser2.deserialize(buffer));
```

Comment:
It would be good to add a check that fails to make sure it returns false but this is all good so I'm merging. Thanks!